jax/experimental/jax2tf/tests/savedmodel_test.py

```diff
@@ -41,25 +42,28 @@ def save_and_load_model(self, model: tf.Module) -> tf.Module:
 
   def test_eval(self):
     f_jax = jax.jit(lambda x: jnp.sin(jnp.cos(x)))
+    signature_type = tf.float64 if config.FLAGS.jax_enable_x64 else tf.float32
```

Comment:
I am sure that this works, but is it needed? It should be possible to save a model that takes f32 arguments in x64 mode.

Reply:
It is possible to save the model. However, unless we explicitly cast `x` to `(j)np.float32`, `jax` will treat it as 64-bit float by default in x64 mode, while the saved model will still load it as a 32-bit float, and `assertAllClose` will fail because of that. We could instead do `x = np.array(0.7, dtype=jnp.float32)` and then have `tf.float32` as our signature type all the time.

Reply:
I think it is better to cast the input.